remoto/backends/__init__.py

```diff
@@ -217,12 +218,12 @@ class JsonModuleExecute(object):
     as well and raised accordingly.
     """
 
-    def __init__(self, conn, module, logger=None):
+    def __init__(self, conn, module, logger=None, remote_python=None):
         self.conn = conn
         self.module = module
         self._module_source = inspect.getsource(module)
         self.logger = logger
-        self.python_executable = None
+        self.python_executable = remote_python
```

Comment:
sine this is referred to as `self.python_executable` can this be made consistent and change from `remote_python` to `python_executable` ?

Reply:
Renamed to `python_executable`.